src/cmake/FindAPR-Util.cmake

```diff
@@ -29,7 +29,7 @@ endmacro(_apu_invoke)
 
 find_program(APR_UTIL_CONFIG_EXECUTABLE
     apu-1-config
-    PATHS /usr/local/bin    /usr/bin    C:/Progra~1/apr-util/bin
+    PATHS /usr/local/bin    /usr/bin    C:/Progra~1/apr-util/bin /usr/local/opt/apr-util/bin
```

Comment:
I suggest grouping non-Windows paths after each other and put Windows last/first. Reads easier this way:

    /usr/local/bin /usr/local/opt/apr-util/bin /usr/bin C:/Progra~1/apr-util/bin 

Besides that, `C:/Progra~1/apr-util/bin` resolves differently depending on where the build is executed: On a 64 Bit system a 64 Bit path is used, while on a 32 Bit system a 32 Bit one is used, because that's the only existing one. Things get interesting if one wants to build 32 Bit on a 64 Bit system, in which case the 64 Bit path would be used in theory.

Is that intended or something to make more clear using an absolute path or simply to ignore for now or ...?

    07.09.2020  10:23    <DIR>          PROGRA~1     Program Files
    07.09.2020  10:23    <DIR>          PROGRA~2     Program Files (x86)

Apps normally deal with those problems by using environment variables like `%ProgramFiles%`. That resolves differently depending on the architecture of the current OS, the architecture of the current process etc.

Reply:
I'm not sure of the reason behind using Progra~1, that's something that Steven Webb added.  I assume it works if you have things installed globally on your system, but I don't think I've ever built it using the apr-config/apu-config executables.

Grouping the non-windows paths makes sense though, I can fix that easily.

Reply:
@stephen-webb Should we change `C:/Progra~1/apr-util/bin` to become `%ProgramFiles%/apr-util/bin` or does it break for you? Using the environment variable instead of the concrete directory name looks more correct to me.

Reply:
I've successfully tested this PR with the following paths instead of the legacy short dir names and therefore suggest to change it:

    $ENV{ProgramFiles}/apr/bin
    $ENV{ProgramFiles}/apr-util/bin